SPECS/python-async-generator/python-async-generator.spec

```diff
@@ -0,0 +1,102 @@
+# what it's called on pypi
+%global srcname async_generator
+# what it's imported as
+%global libname async_generator
+# name of egg info directory
+%global eggname async_generator
+# package name fragment
+%global pkgname async-generator
+
+%global _description \
+This library generally tries hard to match the semantics of Python 3.6's native\
+async generators in every detail (PEP 525), with additional support for yield\
+from and for returning non-None values from an async generator (under the\
+theory that these may well be added to native async generators one day).
+
+Summary:        Async generators and context managers
+Name:           python-%{pkgname}
+Version:        1.10
+Release:        10%{?dist}
+License:        MIT or ASL 2.0
+URL:            https://github.com/python-trio/async_generator
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://files.pythonhosted.org/packages/ce/b6/6fa6b3b598a03cba5e80f829e0dadbb49d7645f523d209b2fb7ea0bbb02a/%{srcname}-%{version}.tar.gz
```

Comment:
>https://files.pythonhosted.org/packages/ce/b6/6fa6b3b598a03cba5e80f829e0dadbb49d7645f523d209b2fb7ea0bbb02a/%{srcname}-%{version}.tar.gz [](http://example.com/codeflow?start=16&length=135)

Could build it with the [GitHub source](https://github.com/python-trio/async_generator/archive/refs/tags/v1.10.tar.gz)? It feels more reliable to take the tarball from the project's page.